client/hotrod-client/src/main/java/org/infinispan/client/hotrod/configuration/Configuration.java

```diff
@@ -86,12 +85,34 @@ public ExecutorFactoryConfiguration asyncExecutorFactory() {
       return asyncExecutorFactory;
    }
 
-   public Class<? extends FailoverRequestBalancingStrategy> balancingStrategyClass() {
-      return balancingStrategyClass;
+   /**
+    * @deprecated since 9.3
```

Comment:
Do you want this merged after we fork ? Still, you should supply javadocs  on the replacement.

Reply:
My view of 'CR' releases is that PRs should get there only if 'needed' (bugfix etc.) or 'really really wanted'. This is neither, I just want to get this in strictly before 10 so that 10 can remove those ifaces. And I didn't expect this to be reviewed in time for Final :)
If you feel so, I can change these to 9.2, though.